leapp/utils/output.py

```diff
@@ -35,7 +36,7 @@ def print_error(error):
 def report_errors(errors):
     if errors:
         sys.stdout.write(pretty_block("ERRORS", color=Color.red))
-        sys.stderr.write("\n")
+        sys.stdout.write("\n")
```

Comment:
Again, why that change, stderr->stdout?

Reply:
Because everything else in this file is going to stdout (see line above).